app/utils/name-generator.js

```diff
@@ -0,0 +1,655 @@
+export default {
+  /**
+   * Generates a random alliterative name. <adjective> + <animal>.
+   */
+  name: function () {
+    var adjectiveIndex = Math.floor(Math.random() * adjectives.length);
+    var adjective = adjectives[adjectiveIndex];
+    var animalSubset = animals.filter(function (animal) {
+      return animal.toLowerCase()[0] === adjective.toLowerCase()[0];
+    });
+    var animalIndex = Math.floor(Math.random() * animalSubset.length);
+    var animal = animalSubset[animalIndex];
+    return `${adjective.capitalize()} ${animal.capitalize()}`;
+  },
+
+  /**
+   * Generates a random identicon avatar url.
+   */
+  avatarUrl: function () {
+    var randomString = (Math.random() + '').substring(2);
+    return `http://www.gravatar.com/avatar/${randomString}?s=256&default=identicon`;
```

Comment:
Look at you getting fancy with es6. I've been meaning to try some of that stuff out myself